articles/tables.md

```diff
@@ -20,727 +20,191 @@ category: howto
-the analytical clients. **See the [API docs](http://docs.synapse.org/rest/org/sagebionetworks/repo/web/controller/TableExamples.html){:target="_blank"} for an enumeration of the types of queries that can be performed.** Here are a couple of simple examples.
-
-To get all of the columns from a `Table` with id syn3079449, the following query would be used:
-
-````
-SELECT * FROM syn3079449
-````
-
-To get only the two columns called "age" and "gender":
-
-````
-SELECT age, gender FROM syn3079449
-````
-
-To count the number of rows:
-
-````
-SELECT count(*) FROM syn3079449
-````
-
-To get all columns, but only rows where age is greater that 50:
-
-````
-SELECT * FROM syn3079449 WHERE age > 50
-````
-
-To get all columns, but only rows where age is greater that 50 - and sort by treatmentArm:
-
-````
-SELECT * FROM syn3079449 WHERE age > 50 ORDER BY "treatmentArm" ASC
-````
-
-
-# Using Tables
-
- A Schema is defined in terms of Column objects that specify types from the following choices:
-
-{:.markdown-table}
-| \<columnType> |
-| ---- |
-| STRING |
-| DOUBLE|
-| INTEGER |
-| BOOLEAN |
-| DATE |
-| ENTITYID |
-| FILEHANDLEID |
-
-<br/>
-
-For the following code examples, we'll be using the [Wondrous Research Project (syn1901847)](https://www.synapse.org/#!Synapse:syn1901847/){:Target="_blank"}. The .csv file used in the example below can be downloaded [here](https://www.synapse.org/#!Synapse:syn7256188){:target="_blank"}.
-
-
-## Creating a Table
-
-**1. Start with a dataframe, .csv, or .tsv:**
-
-{% tabs %}
-
-{% tab Python %}
-{% highlight python %}
-import synapseclient
-syn = synapseclient.login()
-
-# Convert a .csv to a Pandas DataFrame (this requires that you have Pandas installed)
-import pandas as pd
-df = pd.read_csv('path/to/jazzAlbums.csv', index_col=False)
-{% endhighlight %}
-{% endtab %}
-
-{% tab R %}
-{% highlight r %}
-library(synapser)
-synLogin()
-
-csvFile <- read.csv('path/to/jazzAlbums.csv')
-
-# Convert to a data frame in your session:
-df <- as.data.frame(csvFile)
+Synapse `Tables` are designed to provide users the ability to create web-accessible, sharable, and queryable data. Columns in tables can have a user-specified, structured schema. Users may define table columns to contain common primitive data types (text, numbers, dates, etc.) or references to other Synapse objects (e.g., `Files`). 
 
-{% endhighlight %}
-{% endtab %}
+`Tables` may be queried and edited through both the Synapse web UI as well as through our analytical clients that enable direct access to these data from analysis pipeline code. Unlike most NoSQL systems, the data in Synapse `Tables` is strongly consistent, not eventually consistent. This is an important design consideration for scientific data processing, as analysis on eventually-consistent data sources can limit the types of analysis performed, and may require special coding strategies to ensure reasonable accuracy.
```

Comment:
'may require special coding strategies' - using eventually-consistent sources also requires special coding strategies - i.e., waiting, and maybe planning for timeout failures.